core/modules/widgets/dropzone.js

```diff
@@ -27,14 +27,14 @@ DropZoneWidget.prototype = new Widget();
 Render this widget into the DOM
 */
 DropZoneWidget.prototype.render = function(parent,nextSibling) {
-	var self = this;
```

Comment:
It wasn't used, so I removed it. Is that a statement you always expect to be present?